portal/src/main/webapp/js/src/oncoprint/webgl/oncoprint-bundle.js

```diff
@@ -6476,7 +6476,6 @@ var OncoprintWebGLCellView = (function () {
 	renderAllTracks(this, model);
     }
     OncoprintWebGLCellView.prototype.setTrackGroupOrder = function(model) {
-	clearZoneBuffers(this, model);
```

Comment:
sorry about this, I thought I removed it, maybe it got mixed up in a rebase or something.